eng/tools/generator/cmd/issue/issueCmd.go

```diff
@@ -116,17 +116,17 @@ func (c *commandContext) execute() error {
 	if err != nil {
 		return err
 	}
+	// we first output the config to stdout, then validate it so that the user could always get a usable config
+	// validate the config
+	if err := c.validateConfig(*cfg); err != nil {
+		log.Printf("validate config fail:error(%s)", err.Error())
+	}
```

Comment:
after log, it still output to stdout, is it by design?

Reply:
yes, I suppose to filter out some  invalid requests ,and then output the json